apps/web/src/styles/useColor.test.tsx

```diff
@@ -0,0 +1,68 @@
+import { useColorMode } from "@chakra-ui/react";
+import { renderHook } from "@testing-library/react-hooks";
+import { type PropsWithChildren } from "react";
+
+import { dark, light } from "./colors";
+import { useColor } from "./useColor";
+import { LogoDarkIcon, LogoLightIcon } from "../assets/icons";
+import { UmamiTheme } from "../providers/UmamiTheme";
+
+jest.mock("@chakra-ui/react", () => ({
+  ...jest.requireActual("@chakra-ui/react"),
+  useColorMode: jest.fn(),
+}));
+
+jest.mock("../assets/icons", () => ({
+  LogoDarkIcon: () => "logo-dark-icon",
+  LogoLightIcon: () => "logo-light-icon",
+}));
+
+describe("useColor", () => {
+  const mockUseColorMode = useColorMode as jest.Mock;
+
+  const wrapper = ({ children }: PropsWithChildren) => <UmamiTheme>{children}</UmamiTheme>;
+
+  afterEach(() => {
+    jest.clearAllMocks();
```

Comment:
IIRC jest does that automatically, no?

Reply:
This only happens if `clearMocks: true` in the global config, and in our case it is true, so we really can remove `jest.clearAllMocks();`